src/main/host/process.rs

```diff
@@ -636,6 +638,92 @@ impl Process {
         self.check(host);
     }
 
+    /// Send the signal described in `siginfo` to `process`. `currentRunningThread`
+    /// should be set if there is one (e.g. if this is being called from a syscall
+    /// handler), and NULL otherwise (e.g. when called from a timer expiration event).
+    pub fn signal(
+        &self,
+        host: &Host,
+        current_thread: Option<&ThreadRef>,
+        siginfo: &libc::siginfo_t,
+    ) {
```

Comment:
Update argument names in comment? Also maybe explain what the implication is of not passing a thread in?

Reply:
Done